buildroot/package/linux-headers/Config.in.host

```diff
@@ -60,6 +60,10 @@ config BR2_KERNEL_HEADERS_4_18
 	bool "Linux 4.18.x kernel headers"
 	select BR2_TOOLCHAIN_HEADERS_AT_LEAST_4_18
 
+config BR2_KERNEL_HEADERS_4_19
+	bool "Linux 4.18.x kernel headers"
```

Comment:
Typo? :wink:

Reply:
Sorry, didn't realise you'd already fixed that in a later commit!

Reply:
Yes, I noticed it later and just added it to the next commit.